reo/process_results.py

```diff
@@ -248,6 +250,11 @@ def upfront_capex(self):
                 elif len(cost_list) == 1:
                     upfront_capex += (cost_list[0] or 0) * (chp_size or 0)
 
+            # GHP
+            if len(self.dm["ghp_cost"]) >= 1:
+                # TODO currently this only reports the first option cost
```

Comment:
Looks like a TODO that needs addressed before production?

Reply:
Did the TODO